TMS.Net07.Homework.DaysOfWeek/middle.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.DaysOfWeek
+{
+    class middle
+    {
+        enum DaysOfWeek
+        {
+            ПОНЕДЕЛЬНИК,
+            ВТОРНИК,
+            СРЕДА,
+            ЧЕТВЕРГ,
+            ПЯТНИЦА,
+            СУББОТА,
+            ВОСКРЕСЕНЬЕ
+        }
+         public static void task()
+        {
+            DateTime date = new DateTime();
```

Comment:
Creating a `DateTime` instance requires some resources. However, its value will be overwritten in the 30-st line. So, we can just declare a variable.
```suggestion
            DateTime date;
```

Reply:
done